core/src/main/java/org/bitcoinj/script/Script.java

```diff
@@ -717,7 +717,7 @@ public boolean isSentToMultiSig() {
             }
             // First chunk must be an OP_N opcode too.
             if (decodeFromOpN(chunks.get(0).opcode) < 1) return false;
-        } catch (IllegalStateException e) {
+        } catch (IllegalArgumentException e) {
             return false;   // Not an OP_N opcode.
```

Comment:
If you consider this comment, the intended behaviour was to catch that exception.